gridscale/resource_gridscale_postgresql.go

```diff
@@ -132,6 +132,12 @@ func resourceGridscalePostgreSQL() *schema.Resource {
 				Description: "Maximum CPU core count. The PostgreSQL instance's CPU core count will be autoscaled based on the workload. The number of cores stays between 1 and `max_core_count`.",
 				Optional:    true,
 				Computed:    true,
+				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+					if 1 > v.(int) || v.(int) > 32 {
+						errors = append(errors, fmt.Errorf("%v is not a valid value for number of \"max_core_count\". Valid value should be between 1 and 32\n", v.(int)))
+					}
+					return
+				},
```

Comment:
Are you sure this is correct? I think this needs to be 

```go
func(v interface{}, k string) (ws []string, errors []error) []error {
  // …
  return errors
}
```

Here and other ValidateFunc values above.

Reply:
@bkircher fyi, 
![image](https://user-images.githubusercontent.com/9968099/114310263-98b5ad00-9aea-11eb-85c0-66d070c3692c.png)
![image](https://user-images.githubusercontent.com/9968099/114310276-a834f600-9aea-11eb-96ea-65c48a05b287.png)

Reply:
and [Named return values](https://tour.golang.org/basics/7)

Reply:
:sunglasses: Ahh, learned something new here. Thanks!